networking/network_observability/network-observability-overview.adoc

```diff
@@ -24,7 +24,9 @@ The Network Observability Operator requires the following Operators:
 [id="network-observability-operator"]
 == Network Observability Operator
 
-The Network Observability Operator deploys a monitoring pipeline to collect and enrich the network flows. The eBPF agent, which is deployed as a `daemonset` object, creates the network flows. The network flows are then collected and enriched with the Kubernetes metadata, before storing in Loki.
+The Network Observability Operator deploys a monitoring pipeline to collect and enrich the network flows. The eBPF agent, which is deployed as a `daemonset` object, creates the network flows. The network flows are then collected and enriched with the Kubernetes metadata before storing in Loki. 
```

Comment:
The Network Observability Operator doesn't deploy anything.  It provides the Flow Collector API custom resource definition (CRD).  The user needs to create a flow collector resource for it to deploy the things mentioned.